docs/deployment/customization/application.properties-Reference.md

```diff
@@ -798,3 +798,10 @@ is on by default but can be disabled using the following property.
 //boolean
 enable_study_tags=true|false
 ```
+
+# Add Custom Buttons to data tables
+Custom Buttons can be defined which will conditionally appear in all group comparison data tables (with CopyDownloadControls) to launch a custom URL. This can be used, for example, to launch a software application (that is installed on the user's system) with the data. This configuration can also customize new elements on the Visualize page. It points to a JSON file on the classpath. (See [custom_buttons reference](/deployment/customization/custom_buttons-Reference.md)).
```

Comment:
I noticed an inconsistency. When you are viewing the documentation directly in github, these links (to /deployment) don't work, because the documents actually are under "/docs/deployment/....". Example, 
- https://github.com/cBioPortal/cbioportal/blob/master/docs/deployment/customization/application.properties-Reference.md is the correct URL
- https://github.com/cBioPortal/cbioportal/blob/master/deployment/customization/application.properties-Reference.md is where cross-links try to go (giving a 404 error)

When you view the documentation under "docs.cbioportal.org" the links work, because /docs/ is the root and does not appear in the URL path at all.  Example,
-  https://docs.cbioportal.org/deployment/customization/application.properties-reference/

This also means that if you change the link to "/docs/deployment/..." then it will work in the first scenario, but not in the live scenario.  

Not sure how to fix.  It makes it difficult to test the documentation cross-linking prior to it actually going live.

Reply:
Would always using relative URLs solve the issue (e.g. `./../deployment/`)?

Reply:
Yes, I can confirm that works when viewing document in github.  I can't verify that it works under docs.cbioportal.org but it should. I'll submit a new PR that updates the links in this file to be relative

Reply:
added PR #10938 to address this file and a few others.